deepmd/pt/model/model/__init__.py

```diff
@@ -196,6 +218,7 @@ def get_standard_model(model_params):
         type_map=model_params["type_map"],
         atom_exclude_types=atom_exclude_types,
         pair_exclude_types=pair_exclude_types,
+        preset_out_bias=preset_out_bias,
```

Comment:
Need to confirm here. The type hint shows it accepts a `Optional[Dict[str, np.ndarray]]`, but this PR passes a `Optional[Dict[str, List[Optional[torch.tensor]]]]`.

https://github.com/deepmodeling/deepmd-kit/blob/8394101c60f8418ee3727c512d051816c259c690/deepmd/dpmodel/atomic_model/base_atomic_model.py#L43

Also, `torch.tensor` should not be serialized; `numpy.ndarray` should be stored in `@variables`.

https://github.com/deepmodeling/deepmd-kit/blob/8394101c60f8418ee3727c512d051816c259c690/deepmd/dpmodel/atomic_model/base_atomic_model.py#L230-L241

Reply:
actually np.array is enough. however, the type hint in https://github.com/deepmodeling/deepmd-kit/blob/devel/deepmd/pt/utils/stat.py is a mess. Let me fix.

Reply:
@njzjz see https://github.com/deepmodeling/deepmd-kit/pull/4116/commits/836e6c2bfd21994a7e2f22443f1d3dc79682767c

Reply:
most functions in https://github.com/deepmodeling/deepmd-kit/blob/devel/deepmd/pt/utils/stat.py can be moved to deepmd/utils .  @njzjz what do you think?

Reply:
Good to move